.github/workflows/netlify-edge-functions-test.yml

```diff
@@ -1,20 +1,21 @@
 name: Run tests for netlify edge-functions
 
 on:
-  workflow_dispatch
+  pull_request:
+    types: [opened, reopened, synchronize, ready_for_review]
```

Comment:
Is it necessary to test netlify function this many times on each PR?

Reply:
Are we calling in the actual netlify function while testing? Not sure, maybe @Shurtu-gal might help here.
Is that's not the case, then this is okay, since we do that number of times for jest tests also.

Reply:
The actual netlify function is not called at all. Instead the logic is tested through mocking.
https://github.com/asyncapi/website/blob/7b23c07a1c986fc63f6878e5bdce9c5b7fbde0f4/netlify/edge-functions/tests/serve-definitions.test.ts#L54-L76

Reply:
Thanks for clarification @Shurtu-gal 
@akshatnema I think we can then go for running netlify function tests for every PR this number of times. Afterall, they are taking around 11 seconds